src/lib/translate-message-format-compiler.spec.ts

```diff
@@ -97,6 +97,26 @@ describe("TranslateMessageFormatCompiler", () => {
         compiler.compile(messages.answer, "en-GB")({ obj: { q: 3, a: 42 } })
       ).toBe("Answer: 42");
     });
+
+    it("should respect passed-in currency value", () => {
+      const message = "Loan amount: {X, number, currency}";
+
+      compiler = new TranslateMessageFormatCompiler({
+        currency: "USD",
+        formatters: {},
```

Comment:
I guess this won't be necessary anymore once your other PR is merged—let's merge that one first and then it would be great if you could drop `formatters: {}` from the tests here.

Reply:
Sure ! That's what I thought too :)